vendor/vendor.json

```diff
@@ -192,8 +192,8 @@
 		{
 			"checksumSHA1": "90p9joLcyYEYBCMBY2xF/8eYAAM=",
 			"path": "github.com/keybase/client/go/protocol/keybase1",
-			"revision": "804b64589ed62d7e5530153c4434a2fc0caccbac",
-			"revisionTime": "2017-02-08T16:43:02Z"
+			"revision": "3e0a0841a41e8fa7f2f12c279e901086d55bee64",
```

Comment:
We've already vendored this in, in current master.  So you can drop this vendoring part after a rebase.